src/trg-status-bar.c

```diff
@@ -22,6 +22,7 @@
 #include <glib/gi18n.h>
 #include <glib/gprintf.h>
 #include <gtk/gtk.h>
+#include <pango/pango.h>
```

Comment:
I'm pretty sure `gtk.h` includes `pango.h`

Reply:
It certainly compiles, but I thought the proper thing to do in these situations is to be explicit about includes. Not sure what the standard for gtk programs are, if this isn't recommended it can be removed.

Reply:
Its fine either way. Pango is forever a part of GTKs public API so its a safe assumption but if you find it more clear there is no harm at all.